### ISSUE TITLE
WFC connection unreliable - drops after few minutes

### ISSUE THREAD

**Issue by matejdro** [rafaelvcaetano/melonDS-android#1301]
* **Device model:** Pixel 7 Pro
* **Android version:** 14
* **melonDS version:** 1.9.3 PS

**Problem:**  

WFC appears to be pretty unreliable. Friend and I were playing Advance Wars: Days of Ruin, he was on real 3DS and I was on the melonDS. We were both connected to the same wifi network. After a couple of minutes, I will receive an error that I got disconnected from the WFC. His 3DS does not disconnect, so issue is likely with the emulator.

We are using Kaeru WFC.

**Steps to reproduce (if applicable):**
1. Play a game over WFC for couple of minutes

Is there a way to enable logging or some other way to give more info to help diagnose this?

**Comment by Jacobnordvall** [comment]
I didn't even realize WFC worked??

**Comment by matejdro** [comment]
Yup it works and it actually works very well, until it disconnects.
